app/models/resource.rb

```diff
@@ -0,0 +1,128 @@
+class Resource < ApplicationRecord
+  include Triclops::Resource::Info
+
+  validates :identifier, :location_uri, :width, :height, presence: true
+
```

Comment:
It seems like `featured_region` needs to abide by a specific format. Might be good to have a validation for that, I think ActiveRecord has a regex validation. Integer validations for width and height could also be helpful because they would lead to better error messages.

Reply:
Good idea!  I'll add that.  I'm working on a separate PR now, and for easier merging purposes I'll make sure to put it in there.